### PR TITLE
Release 9 Sept 2020

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] Correct Markdown and formatting
- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] All content reviewed

-----
[View rendered activities/tool-management/docusign.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/docusign.md)
[View rendered activities/tool-management/g-suite.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/g-suite.md)
[View rendered activities/tool-management/index.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/index.md)
[View rendered activities/tool-management/jitsi-meet.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/jitsi-meet.md)
[View rendered activities/tool-management/mailing-lists.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/mailing-lists.md)
[View rendered activities/tool-management/mattermost.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/mattermost.md)
[View rendered activities/tool-management/odoo.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/odoo.md)
[View rendered activities/tool-management/podbean.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/podbean.md)
[View rendered activities/tool-management/streamyard.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/streamyard.md)
[View rendered activities/tool-management/tentoo.md](https://github.com/publiccodenet/about/blob/develop/activities/tool-management/tentoo.md)
[View rendered organization/data-processing-tools.md](https://github.com/publiccodenet/about/blob/develop/organization/data-processing-tools.md)

### CODE REVIEW

File: activities/tool-management/odoo.md
Comment (bvhme) [inline activities/tool-management/odoo.md:20]: This is inconsistent, do we work with OpenWorx yet?

**Review by bvhme — CHANGES_REQUESTED** [review]
See comment about OpusVL/OpenWorx.

File: activities/tool-management/odoo.md
Comment (ElenaFdR) [reply activities/tool-management/odoo.md:20]: cc @demkodo!

Comment (demkodo) [reply activities/tool-management/odoo.md:20]: Hey guys, Opus VL never got back to me re; GDPR, they only had GDPR regulations in place for is using their website, not their hosted services, which is truly abhorrent and another reason I'm glad we are leaving. Indeed, openworx isn't active yet but will be soon, hence "our future service provider". So, a middle road will be not having anything around Odoo and GDPR until the migration is complete. Lemme know.

**Comment by ElenaFdR** [comment]
DO NOT MERGE until #751 is merged.

File: activities/tool-management/g-suite.md
Comment (demkodo) [inline activities/tool-management/g-suite.md:17]: remove "a" after "executive" and before "have" + please add (if deemed a good add-on) in sentence "off-boarding policies are in place + and enforced" to make it even more complete, as writing / having policies is one thing, enforcing it is something else.

File: activities/communication/email-signature-template.md
Comment (bvhme) [inline activities/communication/email-signature-template.md:7]: Is this policy or suggestion? I don't think we'd want to enforce this thus opt for weakening the language.

```suggestion
We can use a standard email signature to clearly convey who is communicating and build trust.
```

File: activities/communication/basic-tweet-template.md
Comment (demkodo) [inline activities/communication/basic-tweet-template.md:15]: I think I see an extra space between "create" and the "a" (warm regards, the nit picker)

**Review by bvhme — CHANGES_REQUESTED** [review]
This needs to be resolved:

1. The new community calls guidance under `activities/communication/community-call-blogpost.md` is generic for all community calls, not just the foundation ones, and thus should live with `activities/community-calls/`

Also, I think centralising tool information doesn't make it any more useful for the users of the tool and/or those looking for instance for privacy information. I don't recommend merging this structure, but consent, I think more is there to be learned and we can improve on it later.

File: activities/tool-management/g-suite.md
Comment (ElenaFdR) [reply activities/tool-management/g-suite.md]: Done!

**Comment by ElenaFdR** [comment]
@bvhme /running-community-call.md is also generic and could be moved. 
That leaves only /community-call.md (which duplicates ../community-call/index) and community-call-invite-template.md. 
Is there any reason to leave any content but the template in /communication?

File: activities/communication/email-signature-template.md
Comment (ElenaFdR) [reply activities/communication/email-signature-template.md:7]: Done.

File: activities/communication/basic-tweet-template.md
Comment (ElenaFdR) [reply activities/communication/basic-tweet-template.md:15]: Done in a separate PR!

**Comment by ElenaFdR** [comment]
@bhme @demkodo Your feedback is fixed in #753.

**Review by bvhme — APPROVED** [review]

**Pull request merged**
